### PR TITLE
Assorted fixes

### PR DESCRIPTION
**Pull request by marceloyuela**
A few things in this PR
1: Added in the syntax property since it was missing before.
2: Fixed the ordering of the cmdlet information
3: Added a basic footer for the site
4: Fixed a bug preventing inputs and outputs from being displayed.

Since the sql and dns cmdlets have been added to master, the next PR will be a large merge so that their information can be reflected on the website as well.

### CODE REVIEW

File: website_v2/gcloud-powershell-website-v2/www/static/css/maia.css [hunk 1]
```diff
@@ -1 +1 @@
...
...om,color-stop(0,#5e8ee4),color-stop(1,#4173c9));background-image:linear-gradient(top,#5e8ee4,#4173c9);border:0;-moz-border-radius:2px;-webkit-border-radius:2px;border-radius:2px;color:#fff;cursor:pointer;display:inline-block;font-family:arial,sans-serif;font-size:13px;font-weight:bold;line-height:1.54;padding:7px 12px;text-align:center;text-shadow:0 -1px 0 rgba(0,0,0,.08)}a.maia-button{color:#fff !important;text-decoration:none}button.maia-button{margin:0;overflow:visible;white-space:normal}button.maia-button::-moz-focus-inner{border:0;padding:0}.maia-button:hover,.maia-button:focus{background-color:#396bbc;background-image:-moz-linear-gradient(top,#5587d7,#396bbc);background-image:-webkit-linear-gradient(top,#5587d7,#396bbc);background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0,#5587d7),color-stop(1,#396bbc));background-image:linear-gradient(top,#5587d7,#396bbc)}.maia-button:active{-moz-box-shadow:inset 0 2px 4px rgba(0,0,0,.24);-webkit-box-shadow:inset 0 2px 4px rgba(0,0,0,.24);box-shadow:inset 0 2px 4px rgba(0,0,0,.24)}.maia-button[disabled],.maia-button.maia-button-disabled{background-color:#a4b9e3;background-image:-moz-linear-gradient(top,#b0c7f0,#a4b9e3);
```
Comment (marceloyuela) [inline website_v2/gcloud-powershell-website-v2/www/static/css/maia.css]: The difference in the maia file is the removal of "Â" before bullet points in the footer. It was unnecessary and detrimental.

Comment (marceloyuela) [inline website_v2/gcloud-powershell-website-v2/www/static/css/maia.css]: (before the line was
 #maia-footer-global li+li:before{content:"Â·";padding:0 3px}* html
now it's
#maia-footer-global li+li:before{content:"·";padding:0 3px}* html
)

File: website_v2/gcloud-powershell-website-v2/www/static/app/referenceController.js [hunk 2]
```diff
@@ -28,4 +28,5 @@ app.controller('ReferenceController', ['$scope', '$routeParams',
         });
         
         this.params = $routeParams;
+        this.order = ['synopsis', 'syntax', 'description', 'examples', 'inputs', 'outputs', 'parameters'];
```
Comment (marceloyuela) [inline website_v2/gcloud-powershell-website-v2/www/static/app/referenceController.js]: This order can change later.

File: website_v2/gcloud-powershell-website-v2/www/static/templates/cmdlet.html [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-            {{key.toUpperCase()}}
-        </h2>
-        <pre>
-            {{value}}
-        </pre>
-    </div>
+  class="information-area">
+  <h1>{{refCtrl.params.cmdlet}}</h1>
+  <!-- Display all applicable information for a cmdlet. 
+    An order has to be imposed, as JSONs are unordered. -->
```
Comment (iantalarico) [inline website_v2/gcloud-powershell-website-v2/www/static/templates/cmdlet.html]: nit: json object are not ordered json arrays are :)

**Comment by marceloyuela** [comment]
Added a link to github on the header bar.

**Comment by iantalarico** [comment]
LGTM, nice job.  One very small nit ;)

**Comment by marceloyuela** [comment]
Fixed the nit

**Comment by iantalarico** [comment]
Still LGTM

**Pull request merged**